src/shims/unix/fd.rs

```diff
@@ -257,6 +257,135 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         interp_ok(())
     }
 
+    fn readv(
+        &mut self,
+        fd_num: i32,
+        iov_ptr: &OpTy<'tcx>,
+        iovcnt: i32,
+        offset: Option<i128>,
```

Comment:
This seems to be always `None`, so please remove the argument.

Reply:
* I’ve removed the `offset` parameter from the `readv` function signature as requested.

* Initially, I had included it to align with the parameter structure of the existing `read` and `write` functions, given their similarity to file operations. Let me know if there’s anything else you’d like adjusted!"

![image](https://github.com/user-attachments/assets/a049c70b-bc11-4193-8878-5565960d5a3f)

Reply:
`offset` will be required in the future for `preadv`, but let's go one step at a time :)